qualityassurance/qastagesformattingguidelines.md

```diff
@@ -0,0 +1,89 @@
+# ♻️ QA Testing Stages, Formatting and GuideLines
+
+This Section is to instruct on when to test, how you should be formatting the bugs raised and guideLines for QA
+
+## Developing with QA in mind
+### Commands
+When you are developing a project, the project might require commands to run and work correctly, these can range from simple set up commands to automatic updating commands, a list of these commands and a brief description of what these commands do will help QA move more smoothly and avoid raising unnecessary questions.
+
+Some projects require certain status or manipulation of the database to update systems and cause it to do the next stage/part of the system to test. We would like these to be formed into commands, QA should not need to touch the database side to test the system, this leads to user error. This will allow the tester to carry out complex tasks without directly modify the database or sensitive areas of the application.
+
+
+### ENV
+Ensure that the 'env.example' is update and remove redundant fields/areas that are not required anymore.
+		
+## QA Stages
+For staging we are following a multi stage testing to enable us to ensure that the customer is getting a bug free and fully functioning system. This will be a merge between 'Agile' and 'Waterfall'.
+										
+### Stage 1 - Development Testing
+This stage is for the developers to test and ensure that there are not obvious bugs and that the code is working as expected before they create a PR (Pull Request). This will ensure that QA is not wasting time on obvious bugs that could have easily been spotted during development and be fix them before the pull request is made.
```

Comment:
```suggestion
This stage is for the developers to test and ensure that there are no obvious bugs and that the code is working as expected before they create a PR (Pull Request). This will ensure that QA is not wasting time on obvious bugs that could have easily been spotted during development and be fixed them before the pull request is made.
```